src/app/components/recommendations/md-tabs.scss

```diff
@@ -1,8 +1,20 @@
 md-content {
   background-color: transparent !important;
   md-tabs {
+    min-height: 200px;
     background: #eee;
-    border: 1px solid #ddd;
+    border: 1px solid $primary-color;
+    md-tabs-wrapper {
+      background: white;
+    }
+    & > md-tabs-wrapper > md-tabs-canvas > md-pagination-wrapper > md-ink-bar {
```

Comment:
This is exception. // there is no other way to target that element// because it's angular material dynamically added element. I spent 20 min trying to target md-ink-bar any other way.
Preliminary, I may delete original rule from angular-material.css. But it seem to me a bad idea to edit third part library.